dispatch/apps/events/models.py

```diff
@@ -68,15 +70,33 @@ def save_image_from_url(self, url):
 
 
 @receiver(post_update, sender=Event)
+def send_approved_email(sender, instance, **kwargs):
+
+    if not instance.is_approved:
```

Comment:
We should add an attribute similar to `is_published_email` to avoid sending multiple emails.